README.md

```diff
@@ -67,3 +67,11 @@ provide a desired ordering.  For example, if you `import DataStructures`
 package](https://github.com/JuliaLang/DataStructures.jl) is
 installed), you can pass `dicttype=DataStructures.OrderedDict` to
 maintain the insertion order of the items in the object.
+
+```julia
+JSON.lower(p::Point2D) = [x.x, x.y]
+```
+
+Define a custom serialization rule for a particular data type. Must return a
+value that can be directly serialized; see help for more details. Do **not**
+extend `JSON._print`; that is a clunky API and is now deprecated.
```

Comment:
Fair enough. I do apologize for being so hostile to `JSON._print`; it was the only option, and this package's documentation is quite sparse, so from a consumer's viewpoint it's a very reasonable solution.